src/main/java/io/opentracing/contrib/global/GlobalTracer.java

```diff
@@ -38,43 +38,26 @@
  * The {@linkplain GlobalTracer} will not attempt to choose between implementations:</li>
  * <li>If no single tracer service is found, the {@link io.opentracing.NoopTracer NoopTracer} will be used.</li>
  * </ol>
+ *
+ * @see io.opentracing.util.GlobalTracer
+ * @deprecated GlobalTracer is now part of OpenTracing-java
  */
-public final class GlobalTracer implements Tracer {
```

Comment:
Ok, just realized that the previous version had a private constructor, so, it's not as bad as I thought. But still, you are changing the class signature, which is a breaking change :-) I'm OK with that, *but*, considering that consumers have no expectation of stability for this version, I'd remove it and keep the "legacy" code to a minimum.